arch/x86_64/libk++/bitmap.cpp

```diff
@@ -0,0 +1,93 @@
+#include "x86_64/libk++/bitmap.h"
+#include <stl/cstdlib/stdio.h>
+#include <stl/cstdlib/cstring.h>
+
+namespace firefly::libkern
+{
+    void Bitmap::init(uint32_t *bitmap, size_t size)
```

Comment:
actual C++ <3

Reply:
Was that Sarcasm / did you want me to write a constructor for it?